tests/slr_integration/Admin/Events/Controller_Test.php

```diff
@@ -0,0 +1,125 @@
+<?php
+
+namespace TEC\Tickets\Seating\Admin\Events;
+
+use tad\Codeception\SnapshotAssertions\SnapshotAssertions;
+use TEC\Common\Tests\Provider\Controller_Test_Case;
+use TEC\Tickets\Seating\Admin\Events\Controller;
+use TEC\Tickets\Seating\Service\Layouts as Layouts_Service;
+use TEC\Tickets\Seating\Meta;
+use Tribe\Tests\Traits\With_Uopz;
+
+class Controller_Test extends Controller_Test_Case {
+	use SnapshotAssertions;
+	use With_Uopz;
```

Comment:
To ensure Tickets Commerce will be active, and the `post` type will be ticket-able (it's not by default), use the `Tribe\Tickets\Test\Traits\With_Tickets_Commerce` trait.